msteams-platform/graph-api/rsc/preapproval-instruction-docs.md

```diff
@@ -0,0 +1,238 @@
+
+<br>
+<details>
+
+<summary><b>Preapproval for chat RSC permissions</b></summary>
+
+Use the following cmdlets to check and modify the current state of your chat RSC configurations.
+
+1. `Get-MgBetaChatRscConfiguration`
+1. `Set-MgBetaChatRscConfiguration -State EnabledForPreApprovedAppsOnly`
+
+</details>
+
+<br>
+<details>
+
+<summary><b>Preapproval for team RSC permissions</b></summary>
+
+Use the following cmdlets to check and modify the current state of your team RSC configurations.
+
+1. `Get-MgBetaTeamRscConfiguration`
+1. `Set-MgBetaTeamRscConfiguration -State EnabledForPreApprovedAppsOnly`
+
+</details>
+
+## Create the preapproval of RSC permissions
+
+You can create a preapproval, which preapproves RSC permissions for certain sensitivity labels within your tenant. This allows you to control the data that RSC enabled apps can access.
+
+In this section, you learn to:
+
+* [Create a preapproval without a sensitivity label](#create-the-preapproval-without-a-sensitivity-label) for use with any sensitivity group.
+* [Create a preapproval with a sensitivity label](#create-the-preapproval-with-a-sensitivity-label) for use with specific data.
+
+To create a preapproval, ensure that you have the following information:
+
+1. Teams App ID.
+1. The RSC permissions associated with the app.
+1. Team RSC permission: The sensitivity label ID associated with the sensitivity label you want to link to the preapproval policy.
+1. Global admin privilege in your tenant.
+
+You can use PowerShell cmdlets to create a preapproval. To create a cmdlet, you must gather the information mentioned above with the `New-MgBetaTeamAppPreApproval` command and specify the type of RSC permissions you want to preapprove in the cmdlet.
+
+When you preapprove an app, the following RSC configurations are available:
+
+|Configuration| Description|
+|---|---|
+|`ResourceSpecificApplicationPermissionsAllowedForChats`|When you create a preapproval for RSC permissions for a chat, use this configuration.|
+|`ResourceSpecificApplicationPermissionsAllowedForTeams`|When you create a preapproval for RSC permissions for a team, use this configuration.|
+
+### Create the preapproval without a sensitivity label
+
+When you create your cmdlet to preapprove, you must use the required RSC configuration. If the app you're preapproving has both chat RSC and team RSC permissions, you can include both the configurations.
+
+The following examples show the various ways to build cmdlets for preapproving apps with different RSC permissions, without specifying a sensitivity label:
+
+<br>
+<details>
+
+<summary><b>Preapprove an app with team RSC permissions</b></summary>
+
+```powershell
+New-MgBetaTeamAppPreApproval -TeamsAppId c626ce8b-6d15-4c07-bfb1-a5fd0bc3c20e -ResourceSpecificApplicationPermissionsAllowedForTeams @('OnlineMeeting.ReadBasic.Group')
```

Comment:
>OnlineMeeting.ReadBasic.Group [](http://example.com/codeflow?start=136&length=29)

lets add a real permission. this one doesnt exist

Reply:
addressed in latest commit